server/utils/matchUtils.test.ts

```diff
@@ -260,28 +259,28 @@ describe('matchUtils', () => {
     })
   })
 
-  describe('encodeBedSearchResult', () => {
-    it('encodes a bed search result to Base64', () => {
-      const bedSearchResult = bedSearchResultFactory.build()
+  describe('encodeSpaceSearchResult', () => {
```

Comment:
I've gone for a different approach on my branch, of replacing this function with a call to createQueryString to generate uncoded query params for everything we want to pass to the form page.

Reply:
Yeah I thought you may have done something different here, I had to make some of these changes to please the compiler but thought I might as well change 'bed' to 'space' whilst here. We can mop up and delete this function once your PR is merged.